docs/metrics/customizing-the-sdk/README.md

```diff
@@ -594,7 +607,12 @@ using OpenTelemetry.Metrics;
 using OpenTelemetry.Resources;
 
 using var meterProvider = Sdk.CreateMeterProviderBuilder()
-    .ConfigureResource(r => r.AddService("MyServiceName"))
+    .ConfigureResource(r => r.AddAttributes(new List<KeyValuePair<string, object>>
```

Comment:
Same as https://github.com/open-telemetry/opentelemetry-dotnet/pull/5185/files#r1428553403 and https://github.com/open-telemetry/opentelemetry-dotnet/pull/5185/files#r1428554264